src/guides/v2.3/security/two-factor-authentication.md

```diff
@@ -84,12 +69,13 @@ bin/magento msp:security:tfa:reset <username> <provider>
 ### Advanced emergency steps
 
 {:.bs-callout-warning}
-These advanced steps require a full understanding of database management and modifications. Exercise caution when making any changes directly to your database.
+These advanced steps require a full understanding of database management and modifications. We advise caution when making any changes directly to your database.
 
-In your database, you can modify the following tables and values to affect 2FA.
+In your database, you can modify the following tables and values to affect and override 2FA.
 
 Table: `core_config_data`
 
+-  `msp/twofactorauth/enabled` - Set to zero to disable 2FA globally.
```

Comment:
If we are going to keep references to the MSP 2fa module we should note that this only works with that version. Maybe true for the other references as well.

Reply:
hmmm... I had removed those references to msp -- they might have been lost in the redux thing. I'll remove them as you had earlier suggested.